### PR TITLE
attempt to fix failing new resource action

### PR DESCRIPTION
**Pull request by jukent**
I tried to add a new resource with the button and templated issue, instead of editing the yaml myself. and the build failed
https://github.com/ProjectPythia/projectpythia.github.io/actions/runs/3913226513/jobs/6688872239

This is my attempt to fix it.

### CODE REVIEW

**Comment by jukent** [comment]
Site build failing "Error: The process '/usr/share/miniconda/condabin/conda' failed with exit code 1"

**Comment by dcamron** [comment]
Seems to be a [known issue](https://github.com/conda-incubator/setup-miniconda/issues/274) that could be resolved by switching to mambaforge a la our cookbook actions. Highlights a need to parallel ProjectPythia/pythia-foundations#339 here.

**Comment by brian-rose** [comment]
Has this been resolved with the recent infrastructure updates?

**Comment by jukent** [comment]
I'll rerun the jobs to check

**Comment by jukent** [comment]
Can someone help me understand the error message in pre-commit:
`file "/tmp/pip-build-env-3oiquzwf/overlay/lib/python3.11/site-packages/poetry/core/factory.py", line 57, in create_poetry
                  raise RuntimeError("The Poetry configuration is invalid:\n" + message)
              RuntimeError: The Poetry configuration is invalid:
                - [extras.pipfile_deprecated_finder.2] 'pip-shims<=0.3.4' does not match '^[a-zA-Z-_.0-9]+$'`

**Comment by ktyle** [comment]
@jukent I get the same error. Going to see if pinning Python <3.11 helps ...

**Comment by ktyle** [comment]
@jukent The problem looks to be in the `.pre-commit-config.yaml` file. I will make a separate PR that addresses this.

```

**Comment by jukent** [comment]
That did it @ktyle !

File: .github/workflows/update-resource-gallery.yaml [hunk 1]
```diff
@@ -92,15 +92,15 @@ jobs:
...
               data = json.load(f)
 
           with open(resource_gallery_file) as f:
-              resource-gallery = yaml.load(f)
+              resource_gallery = yaml.load(f)
 
           with open(resource_gallery_file, 'w') as f:
-              resource-gallery.append(data)
+              resource_gallery.append(data)
               yaml.dump(resurce-gallery, f)
```
Comment (brian-rose) [inline .github/workflows/update-resource-gallery.yaml:104]: should this line also be 
```suggestion
              yaml.dump(resurce_gallery, f)
```

**Review by brian-rose — APPROVED** [review]
Looks good to me!

**Pull request merged**